test/cmd/admin/registry.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/../../..
+source "${OS_ROOT}/hack/lib/init.sh"
+os::log::stacktrace::install
+trap os::test::junit::reconcile_output EXIT
+
+defaultimage="openshift/origin-\${component}:latest"
+USE_IMAGES=${USE_IMAGES:-$defaultimage}
+
+os::test::junit::declare_suite_start "cmd/admin/registry"
+# Test running a registry as a daemonset
+os::cmd::expect_failure_and_text 'oadm registry --daemonset --dry-run' 'does not exist'
+os::cmd::expect_success_and_text "oadm registry --daemonset -o yaml --credentials=${KUBECONFIG}" 'DaemonSet'
+os::cmd::expect_success "oadm registry --daemonset --credentials=${KUBECONFIG} --images='${USE_IMAGES}'"
+os::cmd::expect_success_and_text 'oadm registry --daemonset' 'service exists'
+os::cmd::expect_success_and_text 'oc get ds/docker-registry --template="{{.status.desiredNumberScheduled}}"' '1'
```

Comment:
@pweil- this fails when run in isolation but succeeds when run with all the rest of the tests ... any clue?
```
FAILURE after 0.125s: test/cmd/admin/registry.sh:21: executing 'oc get ds/docker-registry --template="{{.status.desiredNumberScheduled}}"' expecting success and text '1': the output content test failed
Standard output from the command:
0
There was no error output from the command.
```